config/settings/test.yml

```diff
@@ -434,7 +434,7 @@ travel_pay:
     resource: 'resource_id'
     tenant_id: 'tenant_id'
     auth_url: 'https://auth.veis.gov'
-  mock: true
+  mock: false
```

Comment:
to test the sts assertion service. all tests should be mocked or use vcr cassettes anyway